packages/server/lib/util/ci_provider.js

```diff
@@ -135,10 +139,12 @@ const _providerCiParams = () => {
       'BUILD_REPOSITORY_URI',
     ]),
     bamboo: extract([
-      'bamboo.resultsUrl',
-      'bamboo.buildNumber',
-      'bamboo.buildResultsUrl',
-      'bamboo.planRepository.repositoryUrl',
+      'bamboo_buildNumber',
+      'bamboo_buildResultsUrl',
+      'bamboo_planRepository_repositoryUrl',
+      'bamboo_planKey',
```

Comment:
I'm not sure why we would collect `planKey` or `agentId` - these don't relate to any data we need or will show later in the Dashboard. 

The `buildKey` should give any necessary info later about a job that we may need for identification and be sufficient.

Reply:
That sounds good. I erred on the side of collecting to much rather than too little. Will update